servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java

```diff
@@ -130,6 +130,7 @@ enum Format {
 
   @CommandLine.Option(
       names = EXPORT_VERSION,
+      defaultValue = "2",
```

Comment:
nit: `ExportImportConstants.DEFAULT_EXPORT_VERSION` ?

Reply:
You are right 🤦